client/my-sites/sidebar/sidebar.jsx

```diff
@@ -507,15 +507,9 @@ export class MySitesSidebar extends Component {
 		/* eslint-disable wpcalypso/jsx-classname-namespace*/
 		return (
 			<li className="wp-admin">
-				<a
-					onClick={ this.trackWpadminClick }
-					href={ adminUrl }
-					target="_blank"
-					rel="noopener noreferrer"
-				>
+				<a onClick={ this.trackWpadminClick } href={ adminUrl }>
 					<Gridicon icon="my-sites" size={ 24 } />
 					<span className="menu-link-text">{ this.props.translate( 'WP Admin' ) }</span>
-					<Gridicon icon="external" size={ 24 } />
```

Comment:
`ExternalLink` would add this, but I'm unclear as to whether this means "you're navigating outside of Calypso" or "we're going to open a new window."

I believe it's the former and we should keep the icon. `ExternalLink` will handle that.